src/data/set/lattice.lean

```diff
@@ -1377,6 +1377,29 @@ begin
   { intros x hz x' hw, exact ⟨x ⊔ x', hs le_sup_left hz, ht le_sup_right hw⟩ }
 end
 
+lemma sInter_prod_sInter_subseteq (f₁ : set (set α)) (f₂ : set (set β)) :
+  ⋂₀ f₁ ×ˢ ⋂₀ f₂ ⊆ ⋂₀ ((λ C : set α × set β, C.1 ×ˢ C.2) '' f₁ ×ˢ f₂) :=
+λ x hx _ ⟨s, hs, h⟩, h ▸ ⟨hx.1 s.1 hs.1, hx.2 s.2 hs.2⟩
+
+lemma sInter_prod_sInter (f₁ : set (set α)) (f₂ : set (set β)) (h₁ : f₁.nonempty)
+  (h₂ : f₂.nonempty) : ⋂₀ f₁ ×ˢ ⋂₀ f₂ = ⋂₀ ((λ C : set α × set β, C.1 ×ˢ C.2) '' f₁ ×ˢ f₂) :=
```

Comment:
Surely you wouldn't called sets `f` 🤠 
```suggestion
lemma sInter_prod_sInter (S : set (set α)) (T : set (set β)) (hS : S.nonempty) (hT : T.nonempty) :
  ⋂₀ S ×ˢ ⋂₀ T = ⋂₀ ((λ st : set α × set β, st.1 ×ˢ st.2) '' S ×ˢ T) :=
```

Reply:
I think it's `f` for family, but I agree with this suggestion -- it's best to follow the naming conventions in the file, where `S` and `T` are used for `set (set α)`.

Reply:
I think I was originally thinking `f` for finite - until I realised that the lemmas work for non-finite collections too.